src/quo2/components/graph/wallet_graph/view.cljs

```diff
@@ -0,0 +1,66 @@
+  (:require [quo2.theme :as theme]
+            [react-native.linear-gradient :as linear-gradient]
+            [react-native.charts :as charts]
+            [react-native.core :as rn]
+            [quo2.components.graph.wallet-graph.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.graph.wallet-graph.utils :as utils]))
+
+(defn- wallet-graph-internal
+  [{:keys [data state time-frame theme]}]
+  (let [max-data-points (case time-frame
+                          :empty    0
+                          :1-week   7
+                          :1-month  30
+                          :3-months 90
+                          :1-year   365
+                          500)
+        data            (if (and (not= time-frame :empty) (> (count data) max-data-points))
+                          (utils/downsample-data data max-data-points)
+                          data)
+        max-value       (when-not (= time-frame :empty) (utils/find-highest-value data))
+        width           (:width (rn/get-screen))
+        line-color      (if (= state :positive)
+                          (colors/theme-colors colors/success-50 colors/success-60 theme)
+                          (colors/theme-colors colors/danger-50 colors/danger-60 theme))
+        gradient-colors [(colors/alpha line-color 0.1) (colors/alpha line-color 0)]
+        fill-color      (colors/theme-colors colors/white colors/neutral-95)]
+    [linear-gradient/linear-gradient
+     {:colors gradient-colors
+      :start  {:x 0 :y 1}
+      :end    {:x 0 :y 0}
+      :style  style/gradient-background}
+     (if (= time-frame :empty)
+       [rn/view
+        {:accessibility-label :illustration
+         :style               style/illustration}
+        [text/text "Illustration here"]]
+       [rn/view {:accessibility-label :line-chart}
+        [charts/line-chart
+         {:height                  96
+          :width                   (+ width 1)
+          :max-value               max-value
+          :min-value               0
+          :adjust-to-width         true
+          :data                    data
+          :area-chart              true
+          :start-fill-color        fill-color
+          :end-fill-color          fill-color
+          :hide-data-points        true
+          :hide-rules              true
+          :hide-y-axis-text        true
+          :x-axis-indices-height   100
+          :thickness               2
+          :color                   line-color
+          :y-axis-thickness        0
+          :x-axis-thickness        0
+          :initial-spacing         0
+          :end-spacing             0
+          :disable-scroll          true
+          :y-axis-label-width      0.01
+          :labels-extra-height     -36
+          :x-axis-label-text-style style/x-axis-label-text-style}]])]))
+
+(def wallet-graph (theme/with-theme wallet-graph-internal))
```

Comment:
I think team agreed on naming components as `view` and `view-internal`

Reply:
@OmarBasem is it documented somewhere?

Reply:
I think yes. cc @ilmotta @J-Son89

Reply:
I think this is not yet documented. There was some agreement many months ago (proof of that is that there are many screens using this *view var convention*), but it got lost and we forgot to update the guidelines. The convention wasn't applied as strongly in quo2 though.

I can update the docs for us.

Reply:
Yep I believe that's what happened @ilmotta

Reply:
To reiterate the idea since it's not written anywhere yet. If a namespace is named `view` and it only exports a single component, then it's sufficient to name the var `view` to avoid the redundancy of `[some-component-name/some-component-name arg1 arg2]`.

Reply:
Cool, I have no strong opinions on this - it was something agreed on by the team in Istanbul last November if I remember correctly

Reply:
@ilmotta 
Yes, the code will be more uniform and I agree with your points. I'm not a big fam pf adding conventions for small things🤔

As @J-Son89 said, it has been already agreed by the team, so it's better to follow it 👍

Thanks for being open to discuss and clarifying!